app/build.gradle

```diff
@@ -1,14 +1,15 @@
 plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
+    id 'kotlin-kapt'
 }
 
 android {
     compileSdk 32
 
     defaultConfig {
         applicationId "com.example.wearableapp"
-        minSdk 30
+        minSdk 26
```

Comment:
It's nice to have. but don't have to support previous version in this app so might as well leave 30 as min sdk, up to you